rust/gui-client/docs/intended_behavior.md

```diff
@@ -39,7 +39,7 @@ Best performed on a clean VM
 1. Export the logs
 1. Expect the zip file to start with "firezone_logs_"
 1. Expect `zipinfo` to show a single directory in the root of the zip, to prevent zip bombing
-1. Expect two subdirectories in the zip, "connlib", and "app", each with 3 files, totalling 6 files
+1. Expect two subdirectories in the zip, "connlib", and "app", with 3 and 2 files respectively, totalling 5 files
```

Comment:
I'm not sure why this had 6 files before. I have two hypotheses:
1. I had been clicking on the notification to test the single-instance behavior, that could create a 3rd log file
2. Maybe the deep link used to write a log file and stopped recently

The 5 files should be correct now though:
1. The IPC service runs when it's first installed
2. After you reboot, the IPC service runs again
3. Then the GUI runs
4. Then there's another reboot, and the IPC service runs again
5. Then the GUI runs again

A 6th file would have to come from somewhere odd.